### ISSUE TITLE
[capture] Crash in Sascha's geometry shader sample

### ISSUE THREAD

**Issue by bradgrantham-lunarg** [LunarG/gfxreconstruct#1472]
Using --swapchain offscreen --offscreen-swapchain-frame-boundary on Android
[Uploading geometry_shader10_20.zip…]()

**Comment by bradgrantham-lunarg** [comment]
[geometry_shader10_20.zip](https://github.com/LunarG/gfxreconstruct/files/14791749/geometry_shader10_20.zip)

**Comment by bradgrantham-lunarg** [comment]
@fabian-lunarg I've added you to this to take a look.  Mostly since you have desktop so far I'd like to know if you can duplicate a crash in geometryshader with "--swapchain offscreen --offscreen-swapchain-frame-boundary" on desktop.  Just more data.

**Comment by bradgrantham-lunarg** [comment]
@yaman-lunarg since you have S23 running GFXR, would you please capture Sascha's geometry shader sample on S23 and see if you encounter a problem replaying it with  `--swapchain offscreen --offscreen-swapchain-frame-boundary`?

**Comment by yaman-lunarg** [comment]
Replay on S23-Ultra:

1. No options replay fails due to lack of `shaderInt64` support
[vanilla.txt](https://github.com/LunarG/gfxreconstruct/files/14950532/vanilla.txt)

2. With `--remove-unsupported`, I observed image on the phone
[removeUnsupported.txt](https://github.com/LunarG/gfxreconstruct/files/14950546/removeUnsupported.txt)

3. With  `--remove-unsupported --swapchain offscreen --offscreen-swapchain-frame-boundary`, I did not observe image on the phone
[requested.txt](https://github.com/LunarG/gfxreconstruct/files/14950550/requested.txt)

**Comment by bradgrantham-lunarg** [comment]
Was that using the "geometry_shader10_20.zip" capture attached above?

**Comment by yaman-lunarg** [comment]
Yes. Would you like me to build Sascha's repo on S23-Ultra?

**Comment by bradgrantham-lunarg** [comment]
Yes, please!  Capture the app on S23-Ultra and replay it and see if you see the same issue.

**Comment by nickdriscoll-lunarg** [comment]
I was able to build, capture, and replay Sascha's geometry shaders sample on my Pixel 8 with `--swapchain offscreen --offscreen-swapchain-frame-boundary`

This `.zip` contains my capture + the screenshots
[geoshader.zip](https://github.com/user-attachments/files/23912754/geoshader.zip)
